src/class-wpml-gutenberg-strings-in-block.php

```diff
@@ -103,7 +114,7 @@ public function update( WP_Block_Parser_Block $block, $string_translations, $lan
 
 		}
 
-		return $block;
+		return apply_filters( 'wpml_update_strings_in_block', $block, $string_translations, $lang );
```

Comment:
@kkarpieszuk, sorry I didn't explicitly asked, but you need to add a PHPDoc here as well.

Reply:
done